airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/BigQueryDestination.java

```diff
@@ -373,23 +373,20 @@ private SerializedAirbyteMessageConsumer getStandardRecordConsumer(final BigQuer
     return new BigQueryRecordStandardConsumer(
         outputRecordCollector,
         () -> {
-          final boolean use1s1t = TypingAndDedupingFlag.isDestinationV2();
```

Comment:
Is it worth deleting the whole `isDestinationV2` method to avoid this kind of accidental resurrection of legacy logic ?. I only see only this 1 usage in IDE, not sure if i can trust that.

Reply:
Good thought - I am gonna punt on that for now since we're still moving more destinations to the v2 format, so I have a feeling we will using again very shortly for redshift, clickhouse, etc.